Library/Formula/corectl.rb

```diff
@@ -28,6 +27,11 @@ def install
     args = []
     args << "VERSION=#{version}" if build.stable?
 
+    if build.head?
+      touch "unreleased"
+      system "git", "add", "unreleased"
+    end
```

Comment:
I just don't understand why you need the `unreleased`; won't it not being a tagged version imply that anyway?